tools/run_tests/performance/dashboard_config/grafana_config/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 gRPC authors.
```

Comment:
nit: why do you have the `_config` suffix after almost all directory names? I think it doesn't add much value.